### PR TITLE
Add anonymize_ip to gtag

### PR DESCRIPTION
**Pull request by albinekb**
https://developers.google.com/analytics/devguides/collection/gtagjs/ip-anonymization

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
           function gtag(){dataLayer.push(arguments);}
           gtag('js', new Date());
 
-          gtag('config', 'UA-84705099-3');
+          gtag('config', 'UA-84705099-3', { 'anonymize_ip': true });
```
Comment (skllcrn) [inline src/index.html:11]: `ga('set', 'anonymizeIp', true);`

**Review by skllcrn — CHANGES_REQUESTED** [review]
In the future we might want to [explore how to avoid setting cookies](https://developers.google.com/analytics/devguides/collection/analyticsjs/cookies-user-id) all together.

File: src/index.html (hunk 1 above)
Comment (albinekb) [reply src/index.html:11]: See the link in the PR https://developers.google.com/analytics/devguides/collection/gtagjs/ip-anonymization, we're using gtag and this is how they specify it there 👍

Comment (skllcrn) [reply src/index.html:11]: Got it! Since we don't use other products than Google Analytics we could move to `analytics.js` for the new site!

**Review by skllcrn — APPROVED** [review]

**Pull request merged**